src/components/sideBar.js

```diff
@@ -148,9 +181,10 @@ export class SideBar extends React.Component {
       <div className="SideBar" >
         <h3>Side Bar!</h3>
         <p>User Avatar</p>
-        <button>USER PROFILE</button>
-        <button>CREATE NEW GAME</button>
-        <button>FIND A GAME</button>
+        <p> test id = {this.props.currentUser.currentUser.id}</p>
```

Comment:
We don't want this appearing in production so just uncomment as needed for your testing